src/utils/DialogMatches.ts

```diff
@@ -41,6 +41,8 @@ export const DialogMatches = {
     UpdateTextMsgSetupDialogMatch: /setup text message/i,
     NotifyDialogMatch: /notify/i,
     PopUpSignInDialogMatch: /signin/i,
+    AdaptiveCardDialog: /adaptive card/i,
```

Comment:
Change to AdaptiveCardDialogMatch - to keep the naming scheme.

Reply:
Done.